forc-test/src/lib.rs

```diff
@@ -74,6 +75,8 @@ pub struct TestResult {
     pub logs: Vec<fuel_tx::Receipt>,
     /// Gas used while executing this test.
     pub gas_used: u64,
+    /// EcalState of the execution
+    pub ecal: Box<EcalState>,
```

Comment:
maybe can use reference or arc here?

Reply:
This is inside `TestResult`, which means the `EcalState` is not shared between tests.